.github/workflows/ci-linux-osx-win-conda.yml

```diff
@@ -29,10 +29,6 @@ jobs:
             compiler: clang-cl
           - name: windows-latest
             os: windows-latest
-          - name: macos-latest
-            os: macos-latest
-            build_type: Release
-            cxx_std: 14
           - name: macos-latest
```

Comment:
it means we will no longer support c++14 ?

Reply:
The CI tests the Python bindings (which now require C++17) so no C++14 testing in CI unless we only test the C++ API

Reply:
mm, then we maybe should provide this setup to our ci. so build without bindings but c++ 14 and then run the tests